lib/fog/vsphere/requests/compute/create_vm.rb

```diff
@@ -255,7 +263,7 @@ def create_controller(controller = nil, index = 0)
             operation: options[:operation],
             device: controller_class.new(key: options[:key] || (1000 + index),
                                          busNumber: options[:bus_id] || index,
-                                         **(options[:type] == RbVmomi::VIM::VirtualAHCIController ? {} : {sharedBus: controller_get_shared_from_options(options)}))
+            **device_options)
```

Comment:
The indenting looks off here

Reply:
No, it's fine I had cross checked

Reply:
This looks off if I look at the file, could it be your editor? https://github.com/fog/fog-vsphere/blob/9b71befc89ddf26021daadfdf86f7c1b3c2be6d1/lib/fog/vsphere/requests/compute/create_vm.rb#L266

Reply:
Updated with a new indentation. Either way, was accepted by rubocop and the general editor format